ui-src/src/app/configurations/configurations.component.ts

```diff
@@ -144,16 +147,29 @@ export class ConfigurationsComponent implements OnInit {
   }
 
   onSubmit(configData) {
-    this.automatedUIServices.postConfig$(configData).subscribe(
+    this.automatedUIServices.postConfig$(configData, this.configPath).subscribe(
       data => {
         this.toastr.success("Your data has been save!", "Success!");
       },
-      error => {
-        this.toastr.error("Failed to save data, Check the Command Line!");
+      error => { switch (error.status) {
+        case 500: {
+          this.toastr.error ("Internal Server Error!");
+                          break;
+                      }
+                      case 400: {
+                       this.toastr.error("Directory Not found!", "An error occurred");
```

Comment:
is this error message accurate all of the time? Perhaps data could be incorrect?

Reply:
in the goLang service, 400 is empty file path or invalid file path. 
(Mike did this code with us when he was down in Galway).

Reply:
"directory not found" is misleading then. It's that the value of a field is invalid, not that the directory cannot be found. "Some of the fields do not conform to the schema" is the boring way of phrasing this. Maybe have a think about a user-friendly message?